playbooks/deprovision.yaml

```diff
@@ -12,81 +12,11 @@
   - fail: msg="Bailing out. this play requires 'THOTH_NAMESPACE'"
     when: THOTH_NAMESPACE is not defined
 
-  - name: "deleting Thoth Templates"
-    command: "oc delete template {{ item }} --namespace {{ THOTH_NAMESPACE }}"
-    with_items:
-      - "thoth-core"
-      - "thoth-core-secret"
-      - "thoth-core-configmap"
-      - "janusgraph"
-      - "result-api-deployment"
-      - "result-api-buildconfig"
-      - "cleanup-buildconfig"
-      - "cleanup-cronjob"
-      - "graph-refresh-buildconfig"
-      - "graph-refresh-cronjob"
-      - "graph-sync-buildconfig"
-      - "graph-sync-cronjob"
-      - "package-extract-buildconfig"
-      - "package-releases-buildconfig"
-      - "package-releases-cronjob"
-      - "cleanup-imagestream"
-      - "graph-refresh-imagestream"
-      - "graph-sync-imagestream"
-      - "package-releases-imagestream"
-    ignore_errors: true
+  - name: Deprovision Thoth
+    command: "oc delete bc,configmap,cronjob,dc,is,pod,role,rolebinding,route,\
+      secret,svc,sa,template,clusterrole --selector 'app=thoth'"
```

Comment:
are we always allowed to delete the clusterrole?!

Reply:
If not, there will be an error message printed and others will (?) be deleted as expected.

Reply:
BTW, this is command from which I got the list (it was not manually curated, I just adjusted some names):

```console
find -iname '*.yaml' -exec grep 'kind:' {} \; | cut -d: -f2 | sort -u | tr '[:upper:]' '[:lower:]'  | tr '\n' ',' | tr -d '[:space:]'
```

Reply:
!